src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml

```diff
@@ -38,7 +38,7 @@ Git: 0.9-fall2024-301-g4e2f0c1c4
     <server init="false" tick="false">true</server>
     <globalAttribute code="0xFFFD" side="either" value="1"/>
     <attribute code="0x0000" side="server" define="INSTALLED_CHIME_SOUNDS" type="array" entryType="ChimeSoundStruct" length="255" minLength="1">InstalledChimeSounds</attribute>
-    <attribute code="0x0001" side="server" define="ACTIVE_CHIME_SOUND_ID" type="int8u" min="0" max="255" default="0" writable="true">ActiveChimeID</attribute>
+    <attribute code="0x0001" side="server" define="ACTIVE_CHIME_SOUND_ID" type="int8u" min="0" max="255" default="0" writable="true">SelectedChime</attribute>
```

Comment:
Doesn't this need changes to chime-server, which now won't compile?  Why do we have dead code that nothing compiles in our tree (which is how this is passing CI)?

Reply:
I would've thought so, but none of the CI failed (I was getting the PR going to find out which CI failed so I could run the commands locally)

Does that mean the cluster server impls don't compile in CI, or at least, the chime server? 😬😬

I'll try to put together a command locally to run it... I am confused as well

Reply:
> Does that mean the cluster server impls don't compile in CI, or at least, the chime server? 😬😬

It means the chime server is not compiled in CI.  Probably because no example app that's compiled in CI actually enables it.

Reply:
That is what I observed too. I had to add the camera-av-stream-mgmt cluster to an example app's zap definition for that server file to compile. So, without a representation of the cluster server impl in any example app, it won't be built. Not just CI, even locally.

Reply:
Addressed - it would seem these were never compiled, there were unrelated compile errors which I fixed